index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="description" content="This is a boilerplate for building websites." />
+    <title>FEWD-DC-15 Boilerplate</title>
+
+    <link rel="stylesheet" href="css/reset.css" type="text/css" />
+    <link rel="stylesheet" href="css/styles.css" type="text/css" />
+  </head>
+  <body>
+
+    <h1>
+      b. musings
+    </h1>
+
+    <h2>
+      i want to muse on...
+    </h2>
+
+    <div class="topics">
+
+      <ul>
+      <li><a style="color: slategrey" href="impinv.html">impact investing...</a></li>
```

Comment:
Avoid inline styles. Bite the bullet and go ahead and put them in the < head> tag or an external css file.